index.js

```diff
@@ -154,40 +153,34 @@ NYC.prototype._readTranspiledSource = function (filePath) {
 }
 
 NYC.prototype.addAllFiles = function () {
-  var _this = this
+  const visitor = relFile => {
```

Comment:
Since this is a one time use function can we keep it in the callback / as an argument to `this.walkAllFiles` but do switch it to arrow function?